js/Grid/Paging.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Paging with Grid widget for Syncfusion Essential JS
-description: How to enable paging and its functionalites.
+title: Paging in JavaScript Grid control | Syncfusion
```

Comment:
Paging in JavaScript Grid Control | Syncfusion

Reply:
changed